Python/bytecodes.c

```diff
@@ -1152,6 +1152,8 @@ dummy_func(
                 PyObject *name = GETITEM(names, oparg);
                 next_instr--;
                 if (_Py_Specialize_StoreAttr(owner, next_instr, name)) {
+                    // "undo" the rewind so end up in the correct handler:
+                    next_instr++;
```

Comment:
Yeah, see my comment at the top of this PR. `_Py_Specialize_StoreAttr` and `_Py_Specialize_LoadAttr` don't really need error handling at all, so the next thing I'm going to do it make them return `void` like the others in 3.12. Then we don't need the branch at all. :)

I'm hesitant to add additional labels and `goto`s to the 3.11 backport, especially since this is pretty cold code. Let me know if you think I should, though.

Reply:
Ok, leave this for this PR.

Maybe make a PR for 3.12 that removes the branch entirely?